### ISSUE TITLE
xmlrpclib.Fault with empty FaultString for opnerp.exceptions.UserError

### ISSUE THREAD

**Issue by mocak** [odoo/odoo#13111]
Impacted versions: Version 9.0c (Community Edition)

Steps to reproduce:

```
class DemoModule:
    __name = "demo.module"
    def  demo_method(self):
        raise UserError("Error message")
        return True

```

From web service api:
`models.execute_kw(db, uid, password,'demo.module', 'demo_method', [[1]])`

Current behavior: xmlrpclib.Fault object with empty FaultStirng

`xmlrpclib.Fault: <Fault 2: 'None'>`

Expected behavior: xmlrpclib.Fault object with FaultStirng "Error message"

Video/Screenshot link (optional):

**Comment by mocak** [comment]
xmlrpc_handle_exception_int method in openerp/service/wsgi_server.py causes this problem. First parameter of UserError is name not value.

**Comment by blaggacao** [comment]
@xmo-odoo @odony Please confirm the fix, that it should be: `e.name` instead...

![image](https://user-images.githubusercontent.com/7548295/34176281-717bf52c-e4cd-11e7-97e4-d6f59e4bc5ce.png)

**Comment by odony** [comment]
Yes, that looks correct, given that we're specifically testing for a UserError :) `e.args[0]` or `e` would work too, but would be less pretty

**Comment by Yenthe666** [comment]
@odony this still isn't handled/fixed. Would you accept a PR to change `e.value` to `e.name`? I'll make the PR then.

**Comment by odony** [comment]
@Yenthe666 sure :)

**Comment by Yenthe666** [comment]
@odony please see https://github.com/odoo/odoo/pull/22276 - we can then forward it to V10 & V11 (where we should change `openerp` to `odoo` for the tools).

**Comment by odony** [comment]
The fix was merged in 9.0 at 5d086cea9b97cec4fe8d07bf6cb06b3d7ee28af7 via #22276. It will be forward-ported soon to later versions. Thanks!
